Crypto/Crypto/Presentation/Views/HomeView/HomeViewModel.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  HomeViewModel.swift
+//  Crypto
+//
+//  Created by Orlando Nicolas Marchioli on 26/10/2023.
+//
+
+import Foundation
+import SwiftUI
+
+class HomeViewModel: ObservableObject{
+    @Environment(\.colorScheme) var colorScheme
+    @Published var isDarkMode = false
+    
+    func toggleDarkMode(){
+        self.isDarkMode.toggle()
+    }
+    
```

Comment:
I think it would be better to have this function in a configuration file that toggles dark mode in all the app, not only home view